src/file-selector.ts

```diff
@@ -108,34 +108,51 @@ function flatten<T>(items: any[]): T[] {
 async function fromDataTransferItem(
   item: DataTransferItem,
   entry?: FileSystemEntry | null,
-) {
-  // Check if we're in a secure context; due to a bug in Chrome (as far as we know) the browser crashes when calling this API (yet to be confirmed as a consistent behavior).
-  //
+): Promise<FileWithPath> {
+  const fileWithPath = await fromFileSystemHandle(item, entry);
+
```

Comment:
I'm really not a fan of these empty lines. It doesn't really improve readability, it just makes adds more eye movement 😆 Please remove them.